modules/registry-configuring-storage-aws-upi.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/registry
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-aws-upi-{context}"]
+= Configuring registry storage for AWS with UPI
+
+During installation, your cloud credentials are sufficient to create a S3 bucket
+and the Registry Operator will automatically configure storage.
+
+If the Registry Operator cannot create an S3 bucket and automatically configure
+storage, you can create a S3 bucket and configure storage with the following
+procedure.
+
+.Prerequisites
+
+* A cluster on AWS with upi
+
+.Procedure
+
+. Create a S3 bucket:
++
+----
+$ oc create configs.imageregistry.operator.openshift.io
```

Comment:
This commands is invalid.

Reply:
When UPI on AWS cluster is ready, s3 storage is automatically configured.

Reply:
@wzheng1 this is only true if the cluster admin/installer provides an AWS account that has permission to create S3 buckets.

Reply:
We should reference AWS docs on how to create an S3 bucket.

Reply:
Whether this doc works? https://docs.aws.amazon.com/AmazonS3/latest/gsg/GetStartedWithS3.html